src/quoridor/game/game.go

```diff
@@ -10,5 +17,7 @@ func CreateGame(conf Configuration) Game {
 	lineCenter := (conf.BoardSize - 1) / 2
 	pawn := Pawn{Position{0, lineCenter}}
 	board := Board{conf.BoardSize}
-	return Game{pawn, board}
+	game := Game{rand.Int(), pawn, board}
```

Comment:
Using a random int is not the best choice because of identifier collisions. Please use a method to check if the identifier does not already exist. 

For example:

var id = myrandid();
do {
    id = myrandid();
} while (checkIdExistInDatabase(id)) ;
return id;

Or, more easily, use a random hash (md5 or wathever else ...) that have more chance to be unique.